docs/add-ins/use-rest-api.md

```diff
@@ -112,4 +109,5 @@ function getCurrentItem(accessToken) {
 
 ## See also
 
-For an example that calls the REST APIs from an Outlook add-in, see [command-demo](https://github.com/OfficeDev/outlook-add-in-command-demo) on GitHub.
+- For an example that calls the REST APIs from an Outlook add-in, see [command-demo](https://github.com/OfficeDev/outlook-add-in-command-demo) on GitHub.
+- Outlook REST APIs are also available through the Microsoft Graph endpoint (see [Outlook REST API via Microsoft Graph](../rest/index.md#outlook-rest-api-via-microsoft-graph)) but there are some key differences, including how your add-in gets an access token. For more details, please [Compare Microsoft Graph and Outlook](../rest/compare-graph.md).
```

Comment:
```suggestion
- Outlook REST APIs are also available through the Microsoft Graph endpoint. For more information, see [Outlook REST API via Microsoft Graph](../rest/index.md#outlook-rest-api-via-microsoft-graph). 
```

Reply:
My suggestion revises this list item for clarity/simplification. The page/section being linked to already references the [Compare Microsoft Graph and Outlook](../rest/compare-graph.md) link, so no need to include it here.